boefjes/boefjes/plugins/kat_ssl_test_ciphers/boefje.json

```diff
@@ -0,0 +1,12 @@
+    {
+        "id": "ssl-test-ciphers",
+        "name": "SSL Test Ciphers",
+        "description": "Run SSL Test Docker container and check ciphers",
+        "consumes": [
+            "IPService"
```

Comment:
This will scan all open ports, not just ports with TLS. I don't think we have a way to only on TLS ports and that isn't a big problem, but we need to handle that testssl.sh will return that there is no TLS. As far as I can see we currently run the normalizer that will return an empty TLSCipher object and then run the bit on that, but I think we should just return an

Reply:
Will check this out!